src/commands/importExportEntities/entities/taxonomies.ts

```diff
@@ -1,11 +1,39 @@
 import { TaxonomyContracts } from "@kontent-ai/management-sdk";
 
+import { zip } from "../../../utils/array.js";
+import { serially } from "../../../utils/requests.js";
 import { EntityDefinition } from "../entityDefinition.js";
 
-export const taxonomiesExportEntity: EntityDefinition<ReadonlyArray<TaxonomyContracts.ITaxonomyContract>> = {
+export const taxonomiesEntity: EntityDefinition<ReadonlyArray<TaxonomyContracts.ITaxonomyContract>> = {
   name: "taxonomies",
   fetchEntities: client => client.listTaxonomies().toAllPromise().then(res => res.data.items.map(t => t._raw)),
   serializeEntities: taxonomies => JSON.stringify(taxonomies),
-  importEntities: () => { throw new Error("Not supported yet.")},
-  deserializeEntities: () => { throw new Error("Not supported yet.")},
+  importEntities: async (client, fileTaxonomies, context) => {
+    const projectTaxonomies = await serially<ReadonlyArray<() => Promise<TaxonomyContracts.ITaxonomyContract>>>(
+      fileTaxonomies.map(taxonomy => () =>
+        client
+          .addTaxonomy()
+          .withData(addExternalIds(taxonomy))
+          .toPromise()
+          .then(res => res.data._raw))
+    );
+
+    return {
+      ...context,
+      taxonomyGroupIdsByOldIds: new Map(zip(fileTaxonomies.map(t => t.id), projectTaxonomies.map(t => t.id))),
+      taxonomyTermIdsByOldIds: new Map(zip(fileTaxonomies.flatMap(t => t.terms), projectTaxonomies.flatMap(t => t.terms)).flatMap(extractTermIdsEntries)),
```

Comment:
I thought it would be a better separation. You never have an id that can be a group or a term. In content types you reference groups in variants you reference terms so why not have them in two maps.